### DISCUSSION TITLE
Welcome to FerrumC Discussions!

### DISCUSSION THREAD

**Discussion by ReCore-sys** [ferrumc-rs/ferrumc#19] [Announcements]
:3

**Comment by hUwUtao** [comment]
:3

**Comment by eetgeenappels** [comment]
:3

**Comment by Sweattypalms** [comment]
👋

**Comment by Skullians** [comment]
no

**Reply by on9au** [reply]
why is original comment in spanish
![image](https://github.com/user-attachments/assets/193fc38f-09be-4f44-9933-0bca78f18e9c)

**Comment by bohdloss** [comment]
:3

**Comment by FedorTaraskin** [comment]
:3

**Comment by JustADude1604** [comment]
so im running the server on a vm running ubuntu server, how do i make offline mode to true and change the operators?
<img width="735" height="324" alt="image" src="https://github.com/user-attachments/assets/b0bc034a-d1c4-486b-9016-c07820c2deb2" />

telling me the directories and the file names would be enough, thanks!

**Reply by ReCore-sys** [reply]
Hey, so the server currently doesn't have permission or an online mode since it's still under development. So technically it's always in offline mode I guess? Keep in mind that quite a lot of features are not yet implemented so it might not be the right fit for your use case right now.
